### PR TITLE
fix: add pass thru for some subdomains

### PR DESCRIPTION
**Pull request by raisedadead**
This will be mostly a temporary workaround for now, to allow for some origins. We should rip this apart in our API migration to LB4.

### CODE REVIEW

**Review by ahmaxed — APPROVED** [review]

File: config/cors-settings.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const whitelistOrigins = [
```
Comment (ojeytonwilliams) [inline config/cors-settings.js]:
```suggestion
exports.whitelistOrigins = [
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

**Comment by scissorsneedfoodtoo** [comment]
I'm getting errors running this locally. @ojeytonwilliams mentioned that we could set `HOME_LOCATION` somewhere.

**Comment by ojeytonwilliams** [comment]
@scissorsneedfoodtoo if you set `HOME_LOCATION='http://localhost:8000'` (same as `sample.env`) it should be fine.

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by scissorsneedfoodtoo** [comment]
Thanks @ojeytonwilliams. This has something to do with my `.env` file. Checked on GitPod and everything seems to be working there.

**Pull request merged**